### PR TITLE
Add send to address

### PR DESCRIPTION
**Pull request by vs4vijay**
New Sub command added to `taker`:

image

Format of `send-to-address` command:

image

### CODE REVIEW

**Comment by vs4vijay** [comment]
Need help with local testing this code, @wthrajat @mojoX911 @rajarshimaitra

File: src/taker/api.rs [hunk 1]
```diff
@@ -1986,4 +1988,26 @@ impl Taker {
         }
         Ok(())
     }
+
+    pub async fn send_to_address(&mut self, rpc_config: RPCConfig, address: String, amount: u64) {
```
Comment (Shourya742) [inline src/taker/api.rs:1992]: Any specific reason to make this async?

File: src/taker/api.rs [hunk 2]
```diff
@@ -1986,4 +1988,26 @@ impl Taker {
...
+                    .expect("Not able to parse address, Please enter correct address"),
+                Amount::from_sat(amount),
+                None,
+                None,
+                None,
+                None,
+                None,
+                None,
+            )
+            .expect("Failed to send transaction");
```
Comment (Shourya742) [inline src/taker/api.rs:2009]: Network::Bitcoin points to the mainnet. In the taker CLI app, the NETWORK is taken as an argument, which can be used to determine the network type.

File: tests/taker_cli.rs [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    // ---- Assertions ----
+
+    // Check the command output
+    assert!(output.status.success(), "Command execution failed");
+
+    // Verify the transaction
+    // let balance = client
+    //     .get_balance(None, None)
+    //     .expect("Failed to get balance");
+    // assert_eq!(balance, amount, "Balance does not match the sent amount");
```
Comment (Shourya742) [inline tests/taker_cli.rs:71]: The CLI app test needs to do the following:
1. It should fund the taker's default wallet, which is created when the taker CLI starts.
2. The test should either accept or be provided with an on-chain address and the amount that needs to be funded by the taker's wallet.

**Review by Shourya742 — CHANGES_REQUESTED** [review]
Concept ACK. Thanks, @vs4vijay, for taking this on. It's great to see BitDev participants showing interest in coinswap. I've added a few comments; let me know if you need any help. Also, feel free to join our Discord for more async discussions: https://discord.gg/fUhJ3HHz

File: src/taker/api.rs (hunk 2 above)
Comment (vs4vijay) [reply src/taker/api.rs:2009]: Sure, in order to test this end to end, would I need to setup any testnet locally? What's the best way to validate this.

Comment (Shourya742) [reply src/taker/api.rs:2009]: Regtest will do the job

**Comment by mojoX911** [comment]
stale and will be overwritten.

**Pull request closed without merging**